src/components/Checkbox/index.tsx

```diff
@@ -61,19 +61,17 @@ export class Checkbox extends React.Component<CheckboxRadioProps> {
     });
   }
 
-  private get textStyle(): StyleProp<TextStyle> {
```

Comment:
The `StyleProp` type includes a lot of fluff that is supported for ingestion, but here you're clearly making a style object